src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,9 @@
+export const ArticleCard = ({ article }) => {
+  return (
+    <div className="article-card">
+      <h3>{article.title}</h3>
+      <img src={article.article_img_url} alt={article.title} />
+      <p>Topic: {article.topic}</p>
+    </div>
```

Comment:
Later on (Ticket 10) we're going to be adding functionality to sort these articles by things such as the date they were written or comment count, it would be good for these things to be visible on the article cards when that happens to give an indication to the user it's worked. Could either add it here now or later, I'll leave it up to you 🙂